services/cloudfront/src/test/java/software/amazon/awssdk/services/cloudfront/CloudFrontUtilitiesIntegrationTest.java

```diff
@@ -89,7 +89,7 @@
 
 public class CloudFrontUtilitiesIntegrationTest extends IntegrationTestBase {
     private static final Base64.Encoder encoder = Base64.getEncoder();
-    private static final String callerReference = Instant.now().toString().substring(0,10);
+    private static final String callerReference = String.valueOf(Instant.now().getEpochSecond());
```

Comment:
Can we borrow this method for this instead? https://github.com/aws/aws-sdk-java-v2/blob/4496c1387b4b937e7303425d32936cd7f095195c/test/service-test-utils/src/main/java/software/amazon/awssdk/testutils/service/S3BucketUtils.java#L68

I know it's from S3, but it should be fine in this case. It's got some traceability built-in by adding the user and test class name that's creating it

Reply:
updated